openrewrite/test/javascript/parser/typeLiteral.test.ts

```diff
@@ -36,6 +36,20 @@ describe('type literal mapping', () => {
           `)
         );
     });
+    test('null save access ?.', () => {
+        rewriteRun(
+            //language=typescript
+            typeScript(`
+                const array: Array<any> = null;
+                console.log(array.length);
+            `),
+            //language=typescript
+            typeScript(`
```

Comment:
I'm not sure about the specifics for typeScript here, but I would expect a method call with two arguments, as opposed to two method calls.
```suggestion
            `,`
```